src/main/java/pro/sky/java/cource2/skyprospringhw4_1/CalculatorImpl.java

```diff
@@ -0,0 +1,69 @@
+package pro.sky.java.cource2.skyprospringhw4_1;
+
+import org.springframework.stereotype.Service;
+
+@Service
+public class CalculatorImpl implements CalculatorService{
```

Comment:
реализацию интерфейса в данном случае лучше именовать как 
```suggestion
public class CalculatorServiceImpl implements CalculatorService{
```